internal/commands/default.go

```diff
@@ -7,13 +7,16 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// version is set during the build process.
+var version = ""
```

Comment:
nit: Here we're assigning an empty string rather than using `var version string`? 😛

Reply:
.. 🤦 . This is quality content.